plugins/catalog-backend-module-github/src/lib/github.ts

```diff
@@ -130,11 +150,19 @@ export async function getOrganizationUsers(
       },
     };
 
-    if (userNamespace) entity.metadata.namespace = userNamespace;
-    if (user.bio) entity.metadata.description = user.bio;
-    if (user.name) entity.spec.profile!.displayName = user.name;
-    if (user.email) entity.spec.profile!.email = user.email;
-    if (user.avatarUrl) entity.spec.profile!.picture = user.avatarUrl;
+    if (item.bio) entity.metadata.description = item.bio;
+    if (item.name) entity.spec.profile!.displayName = item.name;
+    if (item.avatarUrl) entity.spec.profile!.picture = item.avatarUrl;
+
+    if (
+      item.organizationVerifiedDomainEmails &&
+      item.organizationVerifiedDomainEmails.length > 0
+    ) {
+      entity.spec.profile!.email = item.organizationVerifiedDomainEmails[0];
+      if (item.organizationVerifiedDomainEmails.length > 1)
+        entity.metadata.annotations![`${org}/emails`] =
+          item.organizationVerifiedDomainEmails.join(',') as string;
+    } else if (item.email) entity.spec.profile!.email = item.email;
```

Comment:
As I mentioned before, I think it's important for now that the default transformer does NOT change the old behaviour. Therefore, it should probably not mention organizationVerifiedDomainEmails at all, at least not as part of this pull request. Otherwise it'll be a hard surprise for adopters who make a small update to the package, which seems to only add an optional new feature of making custom transformations - but their outputs changed nonetheless behind their backs, perhaps to values that were undesirable to them.

We may still make this behavior the default some time down the line, but not as part of this pull request because it needs consideration and a deprecation period etc.